src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,175 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+
+    private static final int NUMBER_OF_DAYS = 30;
+    private static final int NUMBER_OF_MONTHS = 12;
+    private static final int DAY_START = 4;
+    private static final int DAY_END = 6;
+    private static final int ARRAY_YEAR = 0;
+    private static final int ARRAY_MONTH = 1;
+    private static final int ARRAY_DAY = 2;
+    private static final LocalDate NOW = LocalDate.now();
+    private static final String DATE_FORMAT = "yyyyMMdd";
+    private static final String DATE_TIME_FORMAT = "dd MMMM yyyy HH:mm";
+    private static final String UKRAINE_TIMEZONE = "+02:00";
+
     /**
      * Return the current date as a String depending on a query.
-     *
+     * <p>
      * The query can be passed for the whole date or for it's part:
-     *      - FULL - current date as a whole: year, month, day of month
-     *        formatted as `YYYY-MM-DD` (a default return value);
-     *      - YEAR - current year;
-     *      - MONTH - name of the current month;
-     *      - DAY - current day of month;
+     * - FULL - current date as a whole: year, month, day of month
+     * formatted as `YYYY-MM-DD` (a default return value);
+     * - YEAR - current year;
+     * - MONTH - name of the current month;
+     * - DAY - current day of month;
      * In any other case throw DateTimeException.
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        String result = "";
```

Comment:
you rather use .toString for right return type or String.valueOf

Reply:
I think using `String.valueOf` would look shorter and more uniform for all `cases`, Done)